Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine as builder
+
+RUN apk --update-cache add cmake make g++ boost-dev expat-dev bzip2-dev zlib-dev libpq proj-dev lua5.3-dev postgresql-dev
+ADD . /opt/
+RUN mkdir /opt/build
+WORKDIR /opt/build
+RUN cmake ..
```

Comment:
I used this in my docker-build. Uses LUAJIT and builds release
 cmake -D WITH_LUAJIT=ON -D CMAKE_BUILD_TYPE=MinSizeRel ..